src/main/java/com/softteam/vocabuilder/persistence/entity/Vocabulary.java

```diff
@@ -0,0 +1,33 @@
+package com.softteam.vocabuilder.persistence.entity;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.*;
+import java.util.Date;
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+@Entity
+@Table(name = "vocabularies")
+public class Vocabulary {
+    @Id
+    @Type(type = "uuid-char")
+    @GeneratedValue
+    private UUID id;
+
+    @Column(name = "title",nullable = false)
```

Comment:
I think that no attribute should be able to be nullable, maybe updated yes.

Reply:
what I wanted to say is, by default they are required, or optional? just in case by default they are required, we don't need to set the `nullable = false`

Reply:
The topic is not clear to me.

Reply:
por defecto es `nullable=true`